pages/vi/vi-first-steps.md

```diff
@@ -79,10 +79,10 @@ Follow the tutorial under the [GitHub Issues tab](vi-github-issues.md) to create
 
 Make sure you have created at least one issue, resolved it, commented on an issue you didn't create and have a pull request with the fix merged. You may continue making Issues, Pull requests and Comments, or move on to the next step, while you are waiting for your pull request to be merged. The approval process for your Step 6 fix may take time, but __it is OK__ to continue working.
 
-- Once you complete Step 6 you will have:
-   * 2 pull requests made (one at step 3 and one at step 6)
-   * 1 comment added (on an issue you didn't create)
-   * 1 issue created
+*** <font color=red size=4>- Once you complete Step 6 you will have: </font>*** 
```

Comment:
Try not to use html tags.
Also these points just the summary of step. So not required to highlight. If steps followed properly they will already have all of these points completed.